hedyorg/hedy
# 🖊️ Allow assigning list access to list access

This PR allows in level 16 to assign a list element another list element, e.g. `a[1] = b[2]`

Fixes #5758

**How to test**
Go to level 16 and try the following code:
```
kassa_munten = [1,1,1,1,1,1]
random_aantal_muntjes = [2,3,4,5, 6]
kassa_munten[1] = random_aantal_muntjes[random]
print kassa_munten[1]
```
Ensure that it works without errors
